go/vt/worker/key_resolver.go

```diff
@@ -108,7 +110,7 @@ func newV3ResolverFromColumnList(keyspaceSchema *vindexes.KeyspaceSchema, name s
 // keyspaceID implements the keyspaceIDResolver interface.
 func (r *v3Resolver) keyspaceID(row []sqltypes.Value) ([]byte, error) {
 	v := row[r.shardingColumnIndex]
-	destinations, err := r.vindex.Map(nil, []sqltypes.Value{v})
+	destinations, err := r.vindex.Map(context.TODO(), nil, []sqltypes.Value{v})
```

Comment:
Is this context okay?

Reply:
here the vcursor is nil which means context also does have any meaning here. 
these are functional vindex map call which does not use context.